pkg/operator/ceph/csi/spec.go

```diff
@@ -394,11 +397,22 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		if err != nil {
 			return errors.Wrap(err, "failed to load CephFS plugin template")
 		}
+		if tp.CSILogRotation {
+			tp.CsiComponentName = "node-plugin"
```

Comment:
define const names for node-plugin and controller-plugin and use it in multiple places.